core/ui/src/components/settings/CloudLogManagerConfigureModal.vue

```diff
@@ -202,6 +209,13 @@ export default {
         this.tenant = this.configuration.tenant;
       }
 
+      if (!this.configuration.cluster_id == "") {
+        this.cluster_id =
```

Comment:
This string is not internationalized. Let's use a i18 string with a parameter, i.e. `This cluster is identified as {clusterId} in the Cloud Log Manager UI`